### DISCUSSION TITLE
TSML Plugin Update > v3.13

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#707] [Q&A] [answered]
What was changed in this version?

**Comment by kiyote33** [comment, marked as answer]
Per the changelog:
**3.13**
- Add change detection notification option for feeds
- Update url in TSML UI shortcode
- Fix district dropdown list
- Write approximate value when saving location

**Reply by joshreisner** [reply]
changelogs can be found here https://wordpress.org/plugins/12-step-meeting-list/#developers

**Comment by Grateful-Viv** [comment]
@joshreisner or @kiyote33 Can you elaborate on this shortcode: Update url in TSML UI ?

Thanks.

**Reply by kiyote33** [reply]
The change simply updated the static location where TSML UI loads from: instead of 'react.meetingguide.org' we use our own '**tsml-ui.code4recovery.org'**.

**Reply by Grateful-Viv** [reply]
Which shortcode would I use then?

Have been using [tsml_react]. Should I use  [tsml_ui] instead?

Thanks.

**Reply by kiyote33** [reply]
While we recommend using [tsml_ui], the [tsml_react] will continue to work. It was only the path variable in the code that changed.
